### PR TITLE
Com aggregation doc

### PR DESCRIPTION
**Pull request by jviau**

### CODE REVIEW

File: doc/extensibility/com_aggregation.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+In Visual Studio 2017 a MEF-based extension for COM aggregation has been added to CPS.
+This extension point should be used only when all other CPS extension points are
+insufficient and this is __absolutely__ needed. Also note that this extension does
+not behave 100% like project flavoring and has other limitations. This is explained
+below.
+
+## Extension Point
+
+* Contract: `[Export(ExportContractNames.VsTypes.ProjectNodeComExtension)]`
```
Comment (adrianvmsft) [inline doc/extensibility/com_aggregation.md]: Use CSharp if these snippets render better

File: doc/extensibility/com_aggregation.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* Metadata: `[ComServiceIid(typeof(COMInterface))]`
+  * Where `COMInterface` is the interface you want to export
+  * You can use many
+  * Can also use `[ComServiceIid(typeof(MyType), includeInherited: true)]`
+    * This will recursivley add all interfaces found on `MyType`
+
+## Limitations
+
+1) __Cannot__ override existing CPS implementations of COM interfaces.
```
Comment (adrianvmsft) [inline doc/extensibility/com_aggregation.md]: These bullets render on the same line in github

File: doc/extensibility/com_aggregation.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# COM Aggregation
+
+In Visual Studio 2017 a MEF-based extension for COM aggregation has been added to CPS.
```
Comment (adrianvmsft) [inline doc/extensibility/com_aggregation.md]: Add a Visual Studio 2017 tag

File: doc/extensibility/com_aggregation.md (hunk 1 above)
Comment (jviau) [inline doc/extensibility/com_aggregation.md]: They don't render any differently.

**Pull request merged**